protobuf/model_config.proto

```diff
@@ -130,6 +130,43 @@ message ModelRateLimiter
   uint32 priority = 2;
 }
 
+//@@
+//@@  .. cpp:var:: message SecondaryDevice
+//@@
+//@@     The secondary device to run the model instances on.
+//@@
+message SecondaryDevice
+{
+  //@@
+  //@@  .. cpp:enum:: SecondaryDeviceKind
+  //@@
+  //@@     SecondaryDeviceKind of this instance group.
+  //@@
+  enum SecondaryDeviceKind {
+    //@@    .. cpp:enumerator:: SecondaryDeviceKind::KIND_DLA = 0
+    //@@
+    //@@       This instance group represents instances that have a secondary 
```

Comment:
This is a kind for the secondary device, not the instance group. So just say "An NVDLA core. http://nvdla.org/"
And use KIND_NVDLA.